sdk/python/kfp/components/types/type_utils.py

```diff
@@ -12,18 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Utilities for component I/O type mapping."""
-import inspect
+
 import re
-from typing import Any, List, Optional, Type, Union
+from typing import Any, Optional, Type, Union
 import warnings
 
 import kfp
 from kfp.components import task_final_status
-from kfp.components import v1_structures
 from kfp.components.types import artifact_types
 from kfp.components.types import type_annotations
 from kfp.pipeline_spec import pipeline_spec_pb2
 
+DEFAULT_ARTIFACT_SCHEMA_VERSION = '0.0.1'
```

Comment:
v1 component YAML doesn't include an artifact schema version, so we want to fall back to the behavior we've had for deserializing v1 component YAML so far. Using a constant creates a single source of truth for this value.